README_zh.md

```diff
@@ -0,0 +1,188 @@
+<h1 align="center"> LVGL - Light and Versatile Graphics Library</h1>
+
+<p align="center">
+<img src="https://lvgl.io/assets/images/lvgl_widgets_demo.gif">
+</p>
+<p align="center">
+LVGL provides everything you need to create an embedded GUI with easy-to-use graphical elements, beautiful visual effects and a low memory footprint.
```

Comment:
Is there a reason why it is not translated?

Reply:
In this PR, I leave three places that are not translated into English.
The first place is the title because I need to use 1-2 sentences to introduce the LVGL project and get readers’ interest. I need to think about it carefully.
The second place is CSS. I am not familiar with CSS things.
I will find other Chinese Engineers to discuss how to translate these two places.

The third place is Service. I can translate this part, but there is one word that I cannot understand. Could you tell me what does mean by Kft? or give me a web link to introduce about Kft. I don’t know how to translate Kft this word to Chinese.

In terms of these three places, I will use another PR to translate.

Reply:
Oh, I find it: kft is Korlátolt Felelõsségû Társaság.

Reply:
@kisvegabor all finished